### DISCUSSION TITLE
Testo da scrivere nell'invio di segnalazione

### DISCUSSION THREAD

**Discussion by sistemiinformativicasaspa** [globaleaks/globaleaks-whistleblowing-software#3641] [General]
Buongiorno,
Tempo fa installai una versione figlia di globaleaks per fare il whistleblowing, tale progetto era scaricabile da github all'indirizzo https://github.com/anticorruzione/openwhistleblowing, tuttavia sembrerebbe che al momento che scrivo tale progetto non venga piu' aggiornato, così mi sono installato la vostra versione piu' aggiornata di globaleaks: ho provato ad utilizzarla e c'è un template di segnalazione da utilizzare: come faccio a sapere come customizzare tale template in modo da rispettare la normativa Anac in Italia? Esistono progetti scaricabili piu' aggiornati?

Fatemi sapere,

Francesco Rizzello

**Comment by evilaliv3** [comment]
Gentile utente,

la ringraziamo per la domanda.

Confermiamo che il progetto @openwhistleblowing è attualmente deprecato e ci si aspetta che l'Autorità ne dia presto comunicazione dando indicazioni agli utenti che ancora lo usano; Stiamo infatti partecipando della progettazione del portale dell'Autorità basato oggi sulla versione di GlobaLeaks ufficiale; non sappiamo se l'Autorità continuerà a dare indicazioni specifiche in materia di implementazione.

In merito alle pubbliche amministrazioni che segue vi invitiamo a visionare il progetto WhistleblowingPA (www.whistleblowing.it) in cui abbiamo predisposto un servizio SaaS per le PA.
Potreste valutare di aderire a quello o eventualmente prenderlo a modello.

Posso dirle che a livello di funzionalità il software soddisfa da solo tutti i criteri di legge; in merito agli specifici questionari non ci sono di fatto particolari indicazioni in quanto gli enti hanno la flessibilità per quel che riguarda i dati richiesti al segnalante di poterli realizzare come crendono funzionini meglio nel loro contesto. I vostri legali e DPO sapranno istruirvi su quelli che sono i precisi dettami di legge quali ad esempio l'inserimento di opportune informative di processo.
